include/seastar/http/exception.hh

```diff
@@ -155,7 +155,7 @@ private:
 class unexpected_status_error : public base_exception {
 public:
     unexpected_status_error(http::reply::status_type st)
-        : base_exception("Unexpected reply status", st)
+        : base_exception(fmt::to_string(st), st)
```

Comment:
So now instead of "Unexpected reply status" you see "continue" (or whatever) - is this clearer? Maybe you need to have both the words "unexpected reply status" and "continue"?
Or maybe you're counting on the printout of the exception's **type** to provide some context? Is the exception type always printed clearly?

Reply:
> So now instead of "Unexpected reply status" you see "continue" (or whatever) - is this clearer? Maybe you need to have both the words "unexpected reply status" and "continue"? Or maybe you're counting on the printout of the exception's **type** to provide some context? Is the exception type always printed clearly?

Yes, I rely on exception type. So far it's been shown in logs, e.g. scylladb/scylladb#16133

![image](https://github.com/scylladb/seastar/assets/4498177/e5ce33eb-c02d-4706-b5aa-25726c576742)